### ISSUE TITLE
dicee-flutter is not a valid Dart package name, change to dicee_flutter (underscore, not hyphen)

### ISSUE THREAD

**Issue by xpat** [londonappbrewery/dicee-flutter#10]
.../dicee-flutter$ flutter build web
Running "flutter pub get" in dicee-flutter...
Missing index.html
.../dicee-flutter$ flutter run -d chrome
This application is not configured to build on the web.
To add web support to a project, run `flutter create .`.
.../dicee-flutter$ flutter create .
"dicee-flutter" is not a valid Dart package name.
...
**DO** use `lowercase_with_underscores' for package names.
....

...appBrewery$ mv dicee-flutter/ dicee_flutter/
...dicee_flutter$ flutter create .
Recreating project ....
dicee_flutter.iml (created)
test/widget_test.dart (created)
...

Wrote 35 files.
All done!

.../dicee_flutter$  flutter run -d chrome
Launching lib/main.dart on Chrome in debug mode...
Building application for the web...

**Comment by TheMuellenator** [comment]
Interesting. Can you send me a pull request for this?

**Comment by xpat** [comment]
It happened again on magic-8, etc.  I'm not sure how to do a pull request since the change I'm proposing is just to the name of the repository.

**Comment by iYadavVaibhav** [comment]
It worked, thanks. But why and what was wrong if the parent folder name has hyphen instead of the underscore.

Thank you.

**Comment by ac2epsilon** [comment]
Yes, quite philosofically interesting inherent bug, which guaraneed can not be solved HERE. As soon as sombody solved it, this will happen NOT here. Brute solution is to kill repo at all, but url hardwired into the course lesson. Lets put it into the folder Minus-mistake and it's sticky consequences
